src/main/java/com/google/api/codegen/configgen/transformer/LanguageTransformer.java

```diff
@@ -161,4 +161,21 @@ public String rewrite(String input) {
       return input.replaceAll(pattern, replacement);
     }
   }
+
+  private static class PythonLanguageFormatter implements LanguageFormatter {
+    private static final RewriteRule PYTHON_REWRITE_RULE =
+        new RewriteRule("^google(?!\\.cloud)", "google.cloud");
+
+    @Override
+    public String getFormattedPackageName(String packageName) {
+      packageName = PYTHON_REWRITE_RULE.rewrite(packageName);
+      List<String> names = Arrays.asList(packageName.split("\\."));
+      if (VersionMatcher.isVersion(names.get(names.size() - 1))) {
```

Comment:
Just trying to understand what this does: what's an example where this check would return true?

Reply:
This handles situations where the package name does not end with a major version such as with longrunning. I added a longrunning test to make sure this case is covered.